lib/led-matrix.cc

```diff
@@ -159,7 +161,7 @@ bool RGBMatrix::SetPWMBits(uint8_t value) {
   }
   return success;
 }
-uint8_t RGBMatrix::pwmbits() { return active_->framebuffer()->pwmbits(); }
+uint8_t RGBMatrix::pwmbits() { return pwm_bits_; }
```

Comment:
Because I think it should provide the current setting and not for the active frame.
The active frame could have not the same value than the global value set in the matrix.
Maybe it would be an idea to implement a getter function for the active framecanvas where you can get the information for the active frame.
But here it should return the value currently set to the matrix (normally the same as for the current frame).